x-pack/plugins/reporting/server/export_types/png/png.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { LicenseType } from '@kbn/licensing-plugin/server';
+import { CancellationToken, TaskRunResult } from '@kbn/reporting-common';
+import apm from 'elastic-apm-node';
+import { Writable } from 'stream';
+import {
+  fromEventPattern,
+  mergeMap,
+  finalize,
+  takeUntil,
+  tap,
+  map,
+  Observable,
+  of,
+  lastValueFrom,
+} from 'rxjs';
+import { PngScreenshotOptions, PngScreenshotResult } from '@kbn/screenshotting-plugin/server';
+import { JobParamsPNGDeprecated, TaskPayloadPNG } from './types';
+import { decryptJobHeaders, ExportType, generatePngObservable, getFullUrls } from '../common';
+import { validateUrls } from '../common/validate_urls';
+import {
+  LICENSE_TYPE_TRIAL,
+  LICENSE_TYPE_CLOUD_STANDARD,
+  LICENSE_TYPE_GOLD,
+  LICENSE_TYPE_PLATINUM,
+  LICENSE_TYPE_ENTERPRISE,
+  PNG_JOB_TYPE,
+  REPORTING_TRANSACTION_TYPE,
+  REPORTING_REDIRECT_LOCATOR_STORE_KEY,
+} from '../../../common/constants';
+
+/**
+ * @deprecated
+ * Used for the Reporting Diagnostic
+ */
+export class PngV1ExportType extends ExportType<JobParamsPNGDeprecated, TaskPayloadPNG> {
+  id = 'png';
+  name = 'PNG';
+  jobType = PNG_JOB_TYPE;
+  jobContentEncoding? = 'base64' as const;
+  jobContentExtension = 'png' as const;
+  validLicenses: LicenseType[] = [
+    LICENSE_TYPE_TRIAL,
+    LICENSE_TYPE_CLOUD_STANDARD,
+    LICENSE_TYPE_GOLD,
+    LICENSE_TYPE_PLATINUM,
+    LICENSE_TYPE_ENTERPRISE,
+  ];
+
+  constructor(...args: ConstructorParameters<typeof ExportType>) {
+    super(...args);
+    this.logger = this.logger.get('png-export-v1');
+  }
+
+  // PR 161712 needs these out of reporting
+  public getScreenshots(options: PngScreenshotOptions): Observable<PngScreenshotResult> {
+    return this.startDeps.screenshotting.getScreenshots({
+      ...options,
+      urls: options?.urls?.map((url) =>
+        typeof url === 'string' ? url : [url[0], { [REPORTING_REDIRECT_LOCATOR_STORE_KEY]: url[1] }]
+      ),
+    });
+  }
+
```

Comment:
For this PR, I don't expect there to be a circular dependency issue if we keep this method out. Maybe it'll make sense in a later PR where this code gets moved around, but for now we should not have this method, and below we should use `this.startDeps.reporting.getScreenshots`